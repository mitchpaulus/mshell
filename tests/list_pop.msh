[1 2 3 4] l!
@l pop str wl str wl
@l pop drop pop drop pop str wl
@l pop str wl
