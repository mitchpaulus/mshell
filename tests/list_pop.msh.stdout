Just(4)
[1 2 3]
Just(1)
None
